ts/Core/Axis/GridAxis.ts

```diff
@@ -273,7 +273,8 @@ H.dateFormats.W = function (this: Highcharts.Time, timestamp: number): string {
 
 // First letter of the day of the week, e.g. 'M' for 'Monday'.
 H.dateFormats.E = function (timestamp: number): string {
-    return dateFormat('%a', timestamp, true).charAt(0);
+    const time = new Highcharts.Time(this.options as any);
+    return time.dateFormat('%a', timestamp, true).charAt(0);
```

Comment:
Why can't we use `this.dateFormat()` but we need to create another `Time` instance? Is that a problem with TS or something else? :)